Week1/exercise/w1/index.js

```diff
@@ -17,6 +28,22 @@ console.log('header: ', header);
 // ====================================== //
 
 // BONUS:
+const addTodoButtonRef = document.querySelector("#btn-addTodo");
+console.log("addTodoButtonRef");
+const addTodo = () => {
+  const getTodoListUlRef = document.getElementById("todoList");
```

Comment:
better to define these variables outside this function such that they are only selected once and not every time this function is called